# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/PCookieStore.ipdl
# Commit: 20708fe16d2a
# Full Hash: 20708fe16d2a1f7e85b609bbfe286ec842b620b4
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2024-12-18 04:27:40
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 5: Exempt third-party cookie blocking if the channel is on the exception list. r=cookie-reviewers,bvandersloot,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223384
# ==============================================================================

diff -r 7c4b003d1e48 -r 20708fe16d2a dom/cookiestore/PCookieStore.ipdl
--- a/dom/cookiestore/PCookieStore.ipdl	Tue Dec 17 21:44:37 2024 +0000
+++ b/dom/cookiestore/PCookieStore.ipdl	Tue Dec 17 21:44:37 2024 +0000
@@ -36,6 +36,7 @@
                    bool thirdPartyContext,
                    bool partitionForeign,
                    bool usingStorageAccess,
+                   bool isOn3PCBExceptionList,
                    bool matchName,
                    nsString name,
                    nsCString path,
@@ -46,6 +47,7 @@
                    bool thirdPartyContext,
                    bool partitionForeign,
                    bool usingStorageAccess,
+                   bool isOn3PCBExceptionList,
                    nsString name,
                    nsString value,
                    bool session,
@@ -60,6 +62,7 @@
                       bool thirdPartyContext,
                       bool partitionForeign,
                       bool usingStorageAccess,
+                      bool isOn3PCBExceptionList,
                       nsString name,
                       nsString path,
                       bool partitioned,